app-tasks/dags/aoi-monitoring/find_aoi_projects_to_update.py

```diff
@@ -50,29 +50,23 @@
 def find_aoi_projects_to_update():
     """Find AOI projects to check for updates and push their IDs to xcoms"""
     logger.info('Finding AOI projects to check for updates')
-    # This is check_output to mock shelling out to some scala command
-    # and returning a sequence of UUIDs
-    projects = subprocess.check_output([
-        'echo', 'some-project, ids, here, maybe, many, of-them'
-    ])
-    return {'project_ids': projects.strip().split(', ')}
 
+    bash_cmd = 'java -cp /opt/raster-foundry/jars/rf-batch.jar com.azavea.rf.batch.Main find_aoi_projects'    
+    cmd = subprocess.Popen(bash_cmd, shell=True, stdout=subprocess.PIPE)
+    projects = [""]
+    for line in cmd.stdout:
```

Comment:
@jisantuc `subprocess.check_output` return the whole output, but we need only one line ): mb you can show me a more clean way?